.claude/commands/claw.md

```
@@ -117,46 +106,131 @@ if m:
+if grep -Eq 'gateway connect failed|falling back to embedded|unauthorized:' "$LOGFILE"; then
+  kill "$CLAW_PID" 2>/dev/null || true
+  echo "OpenClaw gateway path failed; refusing to continue via degraded fallback."
+  sed -n '1,80p' "$LOGFILE"
+  exit 1
+fi
+
+echo "Task dispatched to OpenClaw gateway (PID: $CLAW_PID)"
+echo "Log: $LOGFILE"
+echo "Monitor: tail -f $LOGFILE"
+echo "Kill: kill $CLAW_PID"
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)